commands_add.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 type addCmd struct {
-	id     uint64
+	txID   uint64
```

Comment:
I'm on the fence about it. It's a good point and we can carry on discussion in an Issue.

(this would be a multi-repo rename, btw)